$NetBSD: patch-aa,v 1.3 2012/08/28 22:58:09 marino Exp $

The TEA autoconf macros assume that NetBSD is some weird old operating system
that can't handle dots in library names. Using the same values as newer FreeBSD
versions yields a uniform PLIST for the platforms supported by pkgsrc.
It also fixes build on DragonFly.

https://github.com/tDOM/tdom/issues/12

--- configure.orig	2007-08-05 20:47:49.000000000 +0200
+++ configure	2011-12-29 03:13:10.000000000 +0100
@@ -8283,7 +8283,7 @@
 	    CC_SEARCH_FLAGS=""
 	    LD_SEARCH_FLAGS=""
 	    ;;
-	NetBSD-*|FreeBSD-[1-2].*)
+	FreeBSD-[1-2].*)
 	    # NetBSD/SPARC needs -fPIC, -fpic will not do.
 	    SHLIB_CFLAGS="-fPIC"
 	    SHLIB_LD="ld -Bshareable -x"
@@ -8388,7 +8388,7 @@
 	    UNSHARED_LIB_SUFFIX='${TCL_TRIM_DOTS}.a'
 	    TCL_LIB_VERSIONS_OK=nodots
 	    ;;
-	FreeBSD-*)
+	DragonFly-*|NetBSD-*|FreeBSD-*)
 	    # FreeBSD 3.* and greater have ELF.
 	    SHLIB_CFLAGS="-fPIC"
 	    SHLIB_LD="ld -Bshareable -x"
